webserver/webserver.go

```diff
@@ -0,0 +1,70 @@
+package webserver
+
+import (
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"sync"
+	"wedding_service/certificate"
+)
+
+func Start(httpServer *http.Server, httpsServer *http.Server, certPath string, keyPath string) error {
+	httpsServer.TLSConfig = &tls.Config{
+		MinVersion: tls.VersionTLS12, // HTTP/2 requires TLS 1.2 or higher
+		NextProtos: []string{"h2"},   // Enforce HTTP/2
+	}
+
+	if os.Getenv("MODE") == "development" {
```

Comment:
Maybe we can make some enum's (const iota) with these "magic strings"

'"magic strings" typically refer to string literals hardcoded directly into the code, which can lead to maintainability issues and bugs. They are often used as identifiers for data or functionality, but their lack of type safety and potential for duplication can cause problems.'